install.ps1

```diff
@@ -3,67 +3,65 @@
     Remove-Item -r -fo $xhPath
     Remove-Item -r -fo $xhsPath
 }
 
-# xh.exe extraction start.
+# Create dir for result of extraction
+New-Item -ItemType Directory -Path $destdir -Force | Out-Null
 
+# Decompress the zip file to the destination directory
 Add-Type -Assembly System.IO.Compression.FileSystem
-
 $zip = [IO.Compression.ZipFile]::OpenRead($zipfile)
 $entries = $zip.Entries | Where-Object { $_.FullName -like '*.exe' }
+$entries | ForEach-Object { [IO.Compression.ZipFileExtensions]::ExtractToFile($_, $destdir + "\" + $_.Name) }
 
-# Create dir for result of extraction.
-
-New-Item -ItemType Directory -Path $destdir -Force | Out-Null
-
-# Extraction.
-
-$entries | ForEach-Object { [IO.Compression.ZipFileExtensions]::ExtractToFile( $_, $destdir + $_.Name) }
-
-# Free the zipfile.
-
+# Free the zipfile
 $zip.Dispose()
-
 Remove-Item -Path $zipfile
 
-# Copy xh.exe as xhs.exe into bin.
-
+# Copy xh.exe as xhs.exe into bin
 Copy-Item $xhPath $xhsPath
 
-# Add to environment variables.
-
-$p = [System.Environment]::GetEnvironmentVariable('Path', [System.EnvironmentVariableTarget]::User)
-if (!$p.ToLower().Contains($destdir.ToLower())) {
-
-    # Path to "user"/bin.
-
-    Write-Output "Adding $destdir to your Path"
-	
-    $p += "$destdir"
-    [System.Environment]::SetEnvironmentVariable('Path', $p, [System.EnvironmentVariableTarget]::User)
-    $Env:Path = [System.Environment]::GetEnvironmentVariable('Path', [System.EnvironmentVariableTarget]::Machine) + ";" + $p
+# Get version from zip file name.
+$xhVersion = $($zipfilename.trim("xh-v -x86_64-pc-windows-msvc.zip") )
+
+# Inform user where the executables have been put
+Write-Output "xh v$( $xhVersion ) has been installed to:`n - $xhPath`n - $xhsPath"
+
+# Make sure destdir is in the path
+$userPath = [System.Environment]::GetEnvironmentVariable('Path', [System.EnvironmentVariableTarget]::User)
+$machinePath = [System.Environment]::GetEnvironmentVariable('Path', [System.EnvironmentVariableTarget]::Machine)
+
+# If userPath AND machinePath both do not contain bin, then add it to user path
+if (!($userPath -contains $destdir) -and !($machinePath -contains $destdir))
+{
+    Write-Output "PATH did not contain $destdir"
+    
+    # Update userPath
+    $userPath = $userPath.Trim(";") + ";$destdir"
+    
+    # Modify PATH for new windows
+    Write-Output "Adding $destdir to user PATH."
+	[System.Environment]::SetEnvironmentVariable('Path', $userPath, [System.EnvironmentVariableTarget]::User)
+	Write-Output "Done."
 	
-    # Path to xhs.exe.
-
-    Write-Host "PATH environment variable changed (restart your applications that use command line)." -foreground yellow
+    # Instruct how to modify PATH for the current window
+    Write-Output "Execute the following command to make xh usable immediately in PowerShell:"
+    Write-Host "`$Env:Path = `$Env:Path.Trim(`";`") + `";$destdir`"" -foreground yellow
+    
+    Write-Host "Or restart any terminal to refresh its PATH and use xh."
```

Comment:
Can you add a new line before this instruction is outputted?

```
Downloading: xh-v0.16.1-x86_64-pc-windows-msvc.zip
xh v0.16.1 has been installed to:
 - C:\Users\shara\bin\xh.exe
 - C:\Users\shara\bin\xhs.exe

PATH environment variable updated. For changes to take affect, please restart terminal or execute the following command:
$Env:Path = $Env:Path.Trim(";") + ";C:\Users\shara\bin"
```

Reply:
The script now automatically updates the current terminal's PATH, no need to run the PS command. To ensure a dependable experience, it now recommends restarting the IDE instead of running that PS command, as new terminal tabs within IDEs would still inherit the outdated environment with the PS command.